docs/articles/expensify-classic/connections/netsuite/Configure-Netsuite.md

```diff
@@ -60,7 +60,7 @@ This dictates when reimbursable expenses will export, according to your preferre
 
 ## Export Settings for Reimbursable Expenses 
 
-**Expense Reports:** Expensify transactions will export reimbursable expenses as expense reports by default, which will be posted to the payables account designated in NetSuite. 
```

Comment:
@CortneyOfstad Can you confirm please, this sounds right to me, but would be great to get a double check. Thanks!